test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml

```diff
@@ -329,7 +329,43 @@
   <command>cat foo.#.log</command>
   <output regex="no" type="success">&lt;/verbosegc&gt;</output>
  </test>
- 
+
+ <test id="GC Verbose logs command line option contains %s">
```

Comment:
To be more specific: "GC Verbose logs command line option" -> "GC rotating verbose log file name contains..."